lib/matplotlib/axes/_axes.py

```diff
@@ -2770,6 +2772,14 @@ def errorbar(self, x, y, yerr=None, xerr=None,
             every 5-th datapoint will be plotted. The data plot itself still
             shows all data points.
 
+        border : string, optional, default: None
+            A string denoting the desired color of the border of the errorbar.
+            Default, no border is drawn around the errorbar.
+
+        borderwidth : positive integer, optional, default:3
```

Comment:
having conditional kwargs is not a great pattern.  Is there any reasonable way to combine border + boarder width?